server/app/views/components/ProgramCardFactory.java

```diff
@@ -106,7 +112,16 @@ public DivTag renderCard(ProgramCardData cardData) {
         .condWith(
             !adminNoteText.isBlank(),
             p().withClasses("w-3/4", "mb-8", "pt-4", "line-clamp-3", "text-gray-700", "text-base")
-                .with(span("Admin note: ").withClasses("font-semibold"), span(adminNoteText)));
+                .with(span("Admin note: ").withClasses("font-semibold"), span(adminNoteText)))
+        .condWith(
+            showCategories,
+            p(
+                    span("Categories:  ").withClasses("font-semibold"),
+                    iffElse(
+                        programCategoryNames.isEmpty(),
+                        span("None"),
+                        span(programCategoryNames.stream().collect(Collectors.joining(", ")))))
```

Comment:
Nitpick: String.join(", ", programCategoryNames) would be simpler to read

Reply:
I agree, that's easier to read.  Thanks!  Done.